pipeline/testing/_yamlmini.py

```
@@ -0,0 +1,454 @@
+        while i < n:
+            raw = self._lines[i]
+            if raw.strip() == "":
+                collected.append("")
+                i += 1
+                continue
+            indent = len(raw) - len(raw.lstrip(" "))
+            if indent <= key_indent:
+                break
+            collected.append(raw)
+            i += 1
+        # 去掉尾部空行（clip/strip 都忽略纯尾空行）
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)